pareto/utilities/bounding_functions.py

```diff
@@ -0,0 +1,161 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2023 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+# Title: Supporting functions for both Operational and Strategic models
```

Comment:
I think it would be helpful to add a note here that this method adds bounds to the following variables: v_F_Piped, v_F_Trucked, quality.v_Q, and v_F_DiscretePiped with a note on why others variables are not included.

Reply:
We could absolutely include other variables in future versions. The objective was to develop bounds for these three, being the variables involved in nonlinear functions in the MINLP formulation of the strategic model.

Reply:
Added